examples/c/components/http-hello-world/README.md

```diff
@@ -0,0 +1,37 @@
+# HTTP Hello World
+
+This is a simple C Wasm example that responds with a "Hello World" message for each request.
+
+## Prerequisites
+
+- `cargo` 1.75
+- [`wash`](https://wasmcloud.com/docs/installation) 0.27.0
+- `wasmtime` >=19.0.0 (if running with wasmtime)
+- [wasi-sdk](https://github.com/WebAssembly/wasi-sdk)
+- [wit-bindgen](https://github.com/bytecodealliance/wit-bindgen)
+
+## Building
+
+The wash command does not yet support C for building. The variable `WASI_SDK_DIR` defaults to `/opt/wasi-sdk`.
+```bash
+make
+```
```

Comment:
When making the change to run `wash build`, just note the WASI SDK as a dependency